README.md

```diff
@@ -244,9 +244,11 @@ Alternatively, you can execute commands from the MediaWiki root directory:
 
 ## Release notes
 
-### Version 3.0.0 - TBD
+### Version 3.0.0 - 2025-04-04
 
 * Raised minimum MediaWiki version from 1.35 to 1.39
+* Added support for MediaWiki 1.40, 1.41, and 1.43
```

Comment:
What about 1.42? But maybe we can just say something like "for MediaWiki up to 1.43"